Formula/kustomize.rb

```diff
@@ -0,0 +1,31 @@
+class Kustomize < Formula
+  desc "Template-free customization of Kubernetes YAML manifests"
+  homepage "https://github.com/kubernetes-sigs/kustomize"
+  url "https://github.com/kubernetes-sigs/kustomize.git",
+      :tag => "v1.0.6",
+      :revision => "017c4ae0aa19195db2a51ecc5aa82c56a1f1c99b"
+  head "https://github.com/kubernetes-sigs/kustomize.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["CGO_ENABLED"] = "0"
+
+    revision = `git rev-parse HEAD`.strip
+    tag = `git describe --tags`.strip
+    dir = buildpath/"src/github.com/kubernetes-sigs/kustomize"
+    dir.install buildpath.children - [buildpath/".brew_home"]
+    cd dir do
+      system "go", "install", "-ldflags",
+             "-s -X github.com/kubernetes-sigs/kustomize/pkg/commands.kustomizeVersion=#{tag}"\
+             " -X github.com/kubernetes-sigs/kustomize/pkg/commands.gitCommit=#{revision}"
+      bin.install buildpath/"bin/kustomize"
+      prefix.install_metafiles
+    end
+  end
+
+  test do
+    assert_match "KustomizeVersion:", shell_output("#{bin}/kustomize version")
```

Comment:
Any chance of a test that exercises a deeper level of functionality? Thanks!

Reply:
I thought about this, but unfortunately cannot find anything more meaningful.

Reply:
Any chance we could do something like an extremely minimal version of the first [usage example](https://github.com/kubernetes-sigs/kustomize#1-make-a-kustomization-file) (excluding the cluster part) or?

Reply:
Done, check it out.

Reply:
Nice! Great work!